analysis/dataset_definition_pf_data_development.py

```diff
@@ -19,21 +21,31 @@
     "pharm_first_service": ["983341000000102"],
 }
 
-pharmacy_first_codes = [
-    code for codelist in pharmacy_first_dict.values() for code in codelist
+pharmacy_first_event_codes = [
+    code for codelist in pharmacy_first_events_dict.values() for code in codelist
 ]
 
+pharmacy_first_med_codes = (
+    acute_otitis_media_cod
+    + first_impetigo_treatment_cod
+    + infected_insect_bites_cod
+    + shingles_treatment_cod
+    + sinusitis_cod
+    + sore_throat_cod
```

Comment:
See change to codelists.py
```suggestion
    + sore_throat_treatment_cod
```